package.json

```diff
@@ -22,6 +22,7 @@
     "husky": "^6.0.0",
     "jest": "^26.6.3",
     "lint-staged": ">=10",
+    "moonbeam-types-bundle": "^1.2.7",
```

Comment:
is it used anywhere?

Reply:
Yes, by the new tests, not sure why its in the top level. I will move